docs/references/protocol/ledger-data/ledger-entry-types/directorynode.md

```diff
@@ -56,19 +55,34 @@ There are two kinds of Directories:
 ```
 {% /tab %}
 
+{% tab label="NFT Offer Directory" %}
+```json
+{
+    "Flags": 0,
+    "Indexes": [
+        "91E23C060E8D46CBE436382659CF9E31DE3AEB630927D1657D88D033E330F085"
+    ],
+    "LedgerEntryType": "DirectoryNode",
+    "Owner": "rGSte5MQe6smA8ep5rPtKnVJG3JoySzKgs",
+    "RootIndex": "8696AF0E701716A471303F26F6671AF146ACD1ED34597FF10A3874EF2631DCC3",
+    "index": "8696AF0E701716A471303F26F6671AF146ACD1ED34597FF10A3874EF2631DCC3"
+}
```

Comment:
I dont think this example is correct. Here is a correct example, the NFT offer directory always has a `NFTokenID` field
```

{
   "result": {
      "index": "CC45A27DAF06BFA45E8AFC92801AD06A06B7004DAD0F7022E439B3A2F6FA5B5A",
      "ledger_current_index": 310,
      "node": {
         "Flags": 2,
         "Indexes": [
            "83C81AC39F9771DDBCD66F6C225FC76EFC0971384EC6148BAFA5BD18019FC495"
         ],
         "LedgerEntryType": "DirectoryNode",
         "NFTokenID": "000800009988C43C563A7BB35AF34D642990CDF089F11B445EB3DCCD00000132",
         "RootIndex": "CC45A27DAF06BFA45E8AFC92801AD06A06B7004DAD0F7022E439B3A2F6FA5B5A",
         "index": "CC45A27DAF06BFA45E8AFC92801AD06A06B7004DAD0F7022E439B3A2F6FA5B5A"
      },
      "status": "success",
      "validated": false
   }
}
```

Reply:
Thanks, Shawn. Did you get that result sending a DirectoryNode request? I'm not sure why I didn't get a similar result.

Reply:
Yes i got it by requesting `ledger_entry`. I think you may have used the wrong DirectoryNode. Usually, when an offer is created, multiple DirectoryNode could be changed, I think you may have used the wrong one.